src/components/Facility/ConsultationDetails/QuestionnaireResponsesList.tsx

```diff
@@ -271,63 +271,72 @@ function ResponseCard({
       ? properCase(structuredType.replace(/_/g, " "))
       : item.questionnaire?.title || "";
 
-  return (
+  const content = (
```

Comment:
Don't inline, move out as it's own component for performance/component memoization.